.asf.yaml

```diff
@@ -1,6 +1,8 @@
 # https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features#Git.asf.yamlfeatures-CustomsubjectlinesforGitHubevents
 
 github:
+  protected_branches:
+    master: {}
```

Comment:
would it be possible to require PRs for master commits?

eg https://github.com/apache/incubator-pekko/blob/main/.asf.yaml#L45

Reply:
Thanks for the pointer, I'll add that.